### PR TITLE
feat: [Issuer Adapter] Create Presentation with User Data

### LINKED ISSUE #152: [Issuer Adapter] Create Presentation with user data

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #152  

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: test/bdd/cmd/issuer/main.go [hunk 1]
```diff
@@ -0,0 +1,91 @@
+/*
```
Comment (llorllale) [inline test/bdd/cmd/issuer/main.go:1]: tip: I think you could have used `httptest.Server` instead

File: test/bdd/cmd/issuer/main.go [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+		rw.WriteHeader(http.StatusBadRequest)
+	}
+
+	rw.WriteHeader(http.StatusOK)
+
+	logger.Infof("[issuer] req: %s resp=%s", string(msg), prCardVC)
+}
+
+const (
+	prCardVC = `{
```
Comment (llorllale) [inline test/bdd/cmd/issuer/main.go:58]: why not a credit card statement credential?

Comment (rolsonquadras) [reply test/bdd/cmd/issuer/main.go:58]: CreditCard statement context/example is not defined yet. For now, I added PRCard.

File: test/bdd/cmd/issuer/main.go (hunk 1 above)
Comment (rolsonquadras) [reply test/bdd/cmd/issuer/main.go:1]: Linux doen't support DNS resolution to host machine. I was facing issue with calling host port from inside docker container in github actions - https://github.com/docker/for-linux/issues/264

**Pull request merged**